<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinesMainWidget</class>
 <widget class="QWidget" name="LinesMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bresenhamâ€™s</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="y_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-5</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="x_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-5</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="x_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-5</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="BresenhamWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>349</width>
     <height>349</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Point 2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="gridWidth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="remarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="draw_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DDA</string>
   </property>
  </widget>
  <widget class="DDAWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>349</width>
     <height>349</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Point 1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="y_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-5</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DDAWidget</class>
   <extends>QWidget</extends>
   <header>ddawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BresenhamWidget</class>
   <extends>QWidget</extends>
   <header>bresenhamwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
